tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,9 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+
+    assert result == vendor.remove(item)
```

Comment:
Line 50 already does the 'act' part of the test with `result = vendor.remove(item)`

Your assert statement should check that `vendor.inventory` is unchanged and also see what result is. Since the test is test_removing_not_found_is_false, then we should write `assert result == False`